src/neo4j_genai/retrievers/text2cypher.py

```diff
@@ -103,22 +103,22 @@ def __init__(
                     if validated_data.neo4j_schema_model
                     else get_schema(validated_data.driver_model.driver)
                 )
+            else:
+                self.neo4j_schema = ""
 
         except (Neo4jError, DriverError) as e:
             error_message = getattr(e, "message", str(e))
             raise SchemaFetchError(
                 f"Failed to fetch schema for Text2CypherRetriever: {error_message}"
             ) from e
 
-    def get_search_results(
-        self,
-        query_text: str,
-    ) -> RawSearchResult:
+    def get_search_results(self, query_text: str, **kwargs: Any) -> RawSearchResult:
```

Comment:
Can we use a more explicit parameter instead of kwargs? I think it would make our lifes easier if in the future we want to add parameters to this method (and it will prevent having reserved parameter names and so on). So something like:
```
def get_search_results(self, query_text: str, prompt_params: Optional[dict[str, Any]] = None) -> RawSearchResult:
```